src/style.css

```
@@ -0,0 +1,26 @@
+body {
+  font-size: 13px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: lighter;
+  padding: 1em 0;
+  margin: 0;
+}
+
+p,
+ol {
+  margin: 0;
+}
```

You are repeating `margin: 0` three times (in your `h1`, `h2` and `p, ol` selectors).
Don't repeat yourself.
```suggestion
h1 {
  font-size: 17px;
  font-weight: bold;
}

h2 {
  font-size: 15px;
  font-weight: lighter;
  padding: 1em 0;
}

p,
ol,
h1,
h2 {
  margin: 0;
}
```